ci-operator/step-registry/ipi/conf/azurestack/creds/ipi-conf-azurestack-creds-commands.sh

```diff
@@ -13,38 +13,24 @@ fi
 export PATH="$PATH:/usr/local/bin"
 
 CLUSTER_NAME=${NAMESPACE}-${JOB_NAME_HASH}
-RESOURCE_GROUP=${CLUSTER_NAME}
+if [[ $INSTALLATION_TYPE == *"ipi"* ]]; then
```

Comment:
This switch seems too arbitrary to me; or perhaps too tightly coupled to the existing implementation.... Perhaps it would make more sense to switch on the existing of the infra manifest?